index.html

```diff
@@ -12,18 +12,18 @@
     <main id="main">
         <div id="first">
         <h1 id="title">Dr. Norman Borlaug</h1>
-        <p>The man who saved a billion lives</p>
+        <p>The man who saved a billions lives</p>
     </div>
         <div id="img-div">
             <img id="image" src="https://www.w3schools.com/images/picture.jpg" alt="image" class="center">
             <div id="img-caption">
                 Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields -
-                part of his life-long war on hunger.
+                parts of his life-long war on hunger.
             </div>
         </div>
         <div id="tribute-info">
             <h3>
```

Comment:
wow, this is amazing, your CSS selectors are descriptive. I think  You should use a class if you need to use the same selector more than once within a page or a site. An ID is specific to a single element, classes can be assigned to multiple elements on a page or throughout the website.
example : <div class="container"></div>